Dockerfile

```diff
@@ -1,13 +1,12 @@
 FROM python:3.7-slim
 
-RUN pip install --upgrade pip==22.1.2 && \
+RUN pip install --upgrade pip==22.2.2 && \
     pip install poetry
 
 COPY pyproject.toml poetry.lock /code/
 WORKDIR /code
 
-RUN poetry config virtualenvs.create false && \
-    poetry install
+RUN poetry install
```

Comment:
Since you removed this `poetry config virtualenvs.create false` poetry will create a new virtual env for the packages and not install in the system's python. I tried installing in the system python and things failed with the upgrade of pip.

So to fix this either downgrade pip back to what it was.

Or add `poetry run` to the `ENTRYPOINT` further down in the Dockerfile.

Reply:
Currently the gateway auth service does not even start.